komfydb/common/field.h

```diff
@@ -37,6 +38,17 @@ class Field {
 
   virtual void GetValue(std::string& s) const = 0;
 
+  virtual void SetValue(int i) = 0;
```

Comment:
Remove this from the `Field` interface, add these methods to corresponding Field type classes.